src/test/scala/org/graphframes/lib/PageRankSuite.scala

```diff
@@ -88,10 +89,13 @@ class PageRankSuite extends SparkFunSuite with GraphFrameTestSparkContext {
       val pr = prc.run()
       val prInvalid = pr.vertices
         .select("pagerank")
-        .filter(vertexIds.size != _.get(0).asInstanceOf[SparseVector].size)
-      val prInvalidSize = prInvalid.count()
+        .collect()
```

Comment:
The testing input is a small graph, thus calling `collect` won't hurt much. With this the `1.6.x` version compiles.